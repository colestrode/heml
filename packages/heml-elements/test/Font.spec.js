const render = require('@heml/render')
const Font = require('../build/Font').default

test('renders with some attrs', () => {
  const attrs = {
    href: 'https://example.com'
  }

  return expect(render.renderElement(Font, attrs)).toMatchSnapshot()
});
