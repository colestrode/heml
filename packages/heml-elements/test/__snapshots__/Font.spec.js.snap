// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with some attrs 1`] = `"<!--[if !mso]><!--><link href=\\"https://example.com\\" rel=\\"stylesheet\\" type=\\"text/css\\" /><style type=\\"text/css\\">@import url(https://example.com);</style><!--<![endif]-->"`;
